Simplified.xcodeproj/project.pbxproj

```diff
@@ -6468,7 +6446,6 @@
 				CLANG_WARN_NON_LITERAL_NULL_CONVERSION = YES_ERROR;
 				CODE_SIGN_ENTITLEMENTS = Simplified/SimplyE.entitlements;
 				CODE_SIGN_IDENTITY = "iPhone Developer";
-				CURRENT_PROJECT_VERSION = 7;
```

Comment:
weird, I think this was a merge failure from my PR. `17` is actually incorrect because it's related to version 3.6.3, but on this branch we use 3.7.0. Do you mind making sure SimplyE and SimplyE-R2dev targets both have 3.7.0 (7) ?

Reply:
You are right, SimplyE has 3.7.0(17) for some reason, SimplyE-R2dev is 3.7.0(7) though.
Fixed